### ISSUE TITLE
iOS demo does not talk to browser demo

### ISSUE THREAD

**Issue by lzantal** [andyet/SimpleWebRTC#491]
Hi all,

I successfully run the iOS demo on two devices and they talked to each other. Also made the browser demo working between two browsers. 

Now I am trying to make the browser demo talk to the iOS demo but it does not connect at all.
They both connect to the server and I set the same room name but it is almost like they create a separate room instead of joining in.

Laszlo

**Comment by fippo** [comment]
@hjon: do you recall what signaling server the demo talks to?

**Comment by lzantal** [comment]
@fippo I left both of them the default signaling.simplewebrtc.com, what I found out since then is that the iOS demo uses 0.9 socketio, but the browser one uses 1.x 
Wondering if that causes the issue.

**Comment by fippo** [comment]
yes, those are on different servers (the browser one uses sandbox.simplewebrtc.com) which explains the behaviour you're seeing

**Comment by lzantal** [comment]
That explains why I am getting errors when I switch the browser version over to signaling.simplewebrtc.com.
Is there anyway to make them talk to each other?

**Comment by gunsline2** [comment]
Hi @lzantal I had the same problem, so I download this version of javascript and worked fine

https://github.com/andyet/SimpleWebRTC/blob/e26f7a60acfaf0186841d3fc5b84e4e54a0d606e/latest.js

**Comment by gunsline2** [comment]
Just to help have added a fork with an simplewebrtc working with ios https://github.com/gunsline2/SimpleWebRTC

**Comment by gunsline2** [comment]
@fippo I had to update web version, there is a way to make ios project to talk to new web version?

**Comment by gunsline2** [comment]
@fippo I will tell what I did to work.

1 - Downgrade Signalmaster to use sockets 0.9.6 and update all changes made manually, I added xirsys for turn suport (it is a commit of 2 years ago)
2 - Downgrade Simplewebrtc to use sockets 0.9.6 and update all changes made manually (it is a commit of 2 years ago)
3 - I added de signalmaster host for ios app

So it works.

Obs:. Only for the people wants to have Android, web and ios working, it is possible for Android with Crosswalk.

Hope this help, the only cost you would have would be with xirsys after your software is used a lot.
